test/validators/mosaic.validator.spec.ts

```diff
@@ -15,8 +15,9 @@
  * limitations under the License.
  *
  */
-import {expect} from 'chai';
-import {MosaicValidator} from '../../src/validators/mosaic.validator';
+import { expect } from 'chai';
+import { MosaicValidator } from '../../src/validators/mosaic.validator';
+import { MosaicRestrictionTypeValidator } from '../../src/validators/restrictionType.validator';
```

Comment:
The tests added belong to ``restrictionType.validator.spec`` and not to ``mosaic.validator.spec``.